exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -0,0 +1,172 @@
+                "and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the fourth day of Christmas my true love gave to me, four Calling Birds, three French Hens, " +
+                "two Turtle Doves, and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the fifth day of Christmas my true love gave to me, five Gold Rings, four Calling Birds, " +
+                "three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the sixth day of Christmas my true love gave to me, six Geese-a-Laying, five Gold Rings, " +
+                "four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the seventh day of Christmas my true love gave to me, seven Swans-a-Swimming, " +
+                "six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, " +
+                "and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking, " +
+                "seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, " +
+                "three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the ninth day of Christmas my true love gave to me, nine Ladies Dancing, eight Maids-a-Milking," +
+                " seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, " +
+                "three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, nine Ladies Dancing," +
+                " eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, " +
+                "four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the eleventh day of Christmas my true love gave to me, eleven Pipers Piping, " +
+                "ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, " +
+                "six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, " +
+                "and a Partridge in a Pear Tree.\n" +
+                "\n" +
+                "On the twelfth day of Christmas my true love gave to me, twelve Drummers Drumming, " +
+                "eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, " +
+                "seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, " +
+                "three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n";
```

Comment:
I would say; let's break with the other tracks here for the sake of consistency in our own track. I'll add a further note below in response to @jtigger's comments regarding cross-track checks :)